### PR TITLE
libuv: new versions, new url, drop autotools

### PR DESCRIPTION
**Pull request by haampie**

### CODE REVIEW

File: var/spack/repos/builtin/packages/libuv/package.py [hunk 1]
```diff
@@ -5,29 +5,40 @@
 class Libuv(AutotoolsPackage):
     """Multi-platform library with a focus on asynchronous IO"""
     homepage = "https://libuv.org"
-    url = "https://github.com/libuv/libuv/archive/v1.9.0.tar.gz"
+    url = "https://dist.libuv.org/dist/v1.44.1/libuv-v1.44.1-dist.tar.gz"
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/libuv/package.py:8]: Can you add a brief comment stating that the `*-dist` comes with a configure script already generated?

**Review by alalazo — COMMENTED** [review]
Only a minor comment, otherwise LGTM

File: var/spack/repos/builtin/packages/libuv/package.py (hunk 1 above)
Comment (haampie) [reply var/spack/repos/builtin/packages/libuv/package.py:8]: done

**Review by alalazo — APPROVED** [review]
Thanks

**Pull request merged**